demo/demoproject/manage.py

```diff
@@ -11,7 +11,7 @@ def main():
         raise ImportError(
             "Couldn't import Django. Are you sure it's installed and "
             "available on your PYTHONPATH environment variable? Did you "
-            "forget to activate a virtual environment?"
+            "forget to activate a virtual environment?",
```

Comment:
```suggestion
            "forget to activate a virtual environment?"
```

Reply:
If you can that would be great because the multiline is for the string rather than multiline params in that case.

Reply:
I know, but it's still a single argument in a list of arguments. And that's when a lot of Python-people prefer having a comma at the end. Let me dig into whether it is possible to remove it.

Reply:
Yeah, so it is this rule that is the cause: https://github.com/asottile/add-trailing-comma

And it did correctly understand that it is a multiline string. It's just that it always sets a comma after the last function argument, even if there is just one. You can just disable that one if you don't want those commas, but it's quite correct to have them.

Reply:
@Natim look at the last commit of this PR. It removes it.

Reply:
Thank you!